src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreSummary.cs

```diff
@@ -74,13 +74,48 @@ public RestoreSummary(bool success)
             Errors = errors;
         }
 
-        public static void Log(ILogger logger, IEnumerable<RestoreSummary> restoreSummaries, bool logErrors = false)
+        public static void Log(ILogger logger, IReadOnlyList<RestoreSummary> restoreSummaries, bool logErrors = false)
```

Comment:
This is a breaking API change. Overload?

Reply:
Yes good catch